nixos/tests/freshrss.nix

```diff
@@ -0,0 +1,18 @@
+import ./make-test-python.nix ({ lib, pkgs, ... }: {
+  name = "freshrss";
+  meta.maintainers = with lib.maintainers; [ etu ];
```

Comment:
I'm very happy and honored to see this work going on. I'm looking forward to using the module from upstream and migrating my own usage to this!

But I think you @Stunkymonkey should at least co-maintain the tests with me :wink: